torchao/quantization/quant_api.py

```diff
@@ -1509,7 +1520,9 @@ class Int8DynamicActivationInt8WeightConfig(AOBaseConfig):
     layout: Optional[Layout] = PlainLayout()
     act_mapping_type: Optional[MappingType] = MappingType.SYMMETRIC
     weight_only_decode: bool = False
+    granularity: Optional[Union[PerRow, PerTensor]] = PerRow()
```

Comment:
nit: typing needs to be updated to `Optional[Union[Granularity, List[Granularity, Granularity]]]` I think